include/mxnet/lib_api.h

```diff
@@ -1345,7 +1378,7 @@ extern "C" {
     }
 
     OpResource res(cpu_malloc, cpu_alloc, gpu_malloc, gpu_alloc,
-                   cuda_stream, sparse_malloc, sparse_alloc);
+                   cuda_stream, sparse_malloc, sparse_alloc, cpu_states, gpu_states);
```

Comment:
can we rename it here too?

Just dont want "states" to be confused with stateful ops